code/common/debug.cc

```diff
@@ -97,9 +100,25 @@ namespace common::debug
 
     LocalFree(symbols);
 
+  #elif defined(__linux__)
+    // Extract the backtrace to the requested depth
+    // Note that this doesn't give particularly useful results without
+    // the -rdynamic linker option
+    void* frame_stack[depth];
```

Comment:
No VLA here please. They are not part of C++. Use a `std::vector` instead.

Reply:
Addressed in d7b93b543e4353aec4e9af179b70252021566bdf